pkg/deploy/exec_commands.go

```diff
@@ -67,7 +67,7 @@ func GetKeycloakProvisionCommand(cr *orgv1.CheCluster, cheHost string) (command
 		"$realmDisplayName", realmDisplayName,
 		"$keycloakClientId", keycloakClientId,
 		"$keycloakTheme", keycloakTheme,
-		"$cheHost", cheHost,
```

Comment:
Could it be that it was passed as an argument because at the point in time, the status was still not updated with the last changes (http to https for example) ?
Maybe a dumb question, but still asking ;-)

Reply:
I don't think so.
Previously it was
```
	cheHost := instance.Spec.Server.CheHost
	keycloakProvisionCommand := deploy.GetKeycloakProvisionCommand(instance, cheHost)
```

I think it is just some leftovers.